llvm/lib/ProfileData/InstrProfWriter.cpp

```diff
@@ -604,6 +607,31 @@ Error InstrProfWriter::writeImpl(ProfOStream &OS) {
       OS.writeByte(0);
   }
 
+  uint64_t VTableNamesSectionStart = OS.tell();
+
+  // Use an empty string as compressed vtable names and get the necessary
+  // profile format change in place for version 12.
+  // TODO: Store the list of vtable names in InstrProfWriter and use the
+  // real compressed name.
+  std::string CompressedVTableNames;
```

Comment:
Prefer using a dummy string, a lot of the code below will behave differently e.g. the for loop and the padding loop wont execute at all as far as I can tell.